src/lib/runner.ts

```diff
@@ -2,14 +2,16 @@ import type Run from '@lib/run';
 import formatTime from '@lib/util/formatTime';
 
 export default class Runner {
-	private _rounds = new RoundManager();
+	private _rounds: RoundManager;
 
 	constructor(
 		private _run: Run,
 		private _id: string,
 		private _name: string,
 		private _alias?: string
-	) {}
+	) {
+		this._rounds = new RoundManager();
```

Comment:
Switching from inline- to constructor-initialization is not required and should be undone

Reply:
resolved: https://github.com/tametsi/ligas/pull/1/commits/ec83176d8b098346637230f754fca10aa994dab4